docs/user/operator-hub.adoc

```diff
@@ -0,0 +1,253 @@
+:source-highlighter: pygments
+
+# Odo integration with Operator Hub
```

Comment:
Should Odo/odo be capitalized?

Reply:
No, odo should not be capitalized. Good catch Yana.